jms/1.7/modules/ROOT/pages/index.adoc

```diff
@@ -1,287 +1,70 @@
-
-* NEVER: No lookup is done and the destinations are created using the existing JMS session
-* TRY_ALWAYS: First try to find the destination using JNDI, and in case it doesn't exist, create it using the current JMS Session.
-* ALWAYS: If a queue/topic cannot be found via JNDI, fail with a `JMS:DESTINATION_NOT_FOUND` error
-
-For more information regarding the `generic-connection` or `jndi-connection-factory`, see the connector reference docs.
-
-== Set Up the Connection Libraries
+== Prerequisites
 
-NOTE: No matter what type of connection you are using, you always need to configure a library containing the JMS client implementation since the connector is not bound to any particular implementation.
+To use this connector, you must be familiar with:
 
-=== ActiveMQ External Libraries
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
 
-For an ActiveMQ connection, there are three possible libraries that need to be configured depending on your connection settings:
+Before creating an app, you must have access to Anypoint Platform and Anypoint Studio.
 
-* ActiveMQ JMS Client Library:
+== Common Use Cases for the Connector
 
-The `activemq-client` of your choice that provides a valid `org.apache.activemq.ActiveMQConnectionFactory` implementation.
+JMS Connector enables you to:
 
-WARNING: This dependency is always required.
+* Listen for new messages as they arrive to the destination.
+* Publish a new message and send it to the specified destination.
+* Consume a message at any given time in the flow from any given destination.
+* Publish a message to any destination, and then wait for a reply on a different destination.
 
-For example, you can use:
+For examples, see xref:jms-examples.adoc[JMS Connector Examples].
 
-[source,xml,linenums]
-----
-<dependency>
-     <groupId>org.apache.activemq</groupId>
-     <artifactId>activemq-client</artifactId>
-     <version>5.14.4</version>
- </dependency>
-----
+== JMS Messaging Support Models
 
-* ActiveMQ Broker Library:
+JMS supports two models for messaging:
 
-The `activemq-broker` is only required when using an in-memory broker based on the VM transport (which is the one configured by default). This dependency should provide a valid `org.apache.activemq.broker.Broker` implementation.
+* Point-to-point queues
+* Publish and subscribe topics
 
-For example, you can use:
+=== Point-to-Point Queues
 
-[source,xml,linenums]
-----
-<dependency>
-    <groupId>org.apache.activemq</groupId>
-    <artifactId>activemq-broker</artifactId>
-    <version>5.14.4</version>
-</dependency>
-----
+In the point-to-point or queuing model, a sender posts messages to a particular queue and a receiver reads messages from the queue. In this model, the sender knows the destination of the message and posts the message directly to the receiver’s queue.
```

Comment:
The title names "point-to-point queuing model" but the first sentence adds "or": "point-to-point or queuing model."
Are we naming one model or two?

Reply:
"and a receiver reads messages from _that_ queue"? (the same queue sent to)

Reply:
```suggestion
In the point-to-point queuing model, a sender posts messages to a particular queue and a receiver reads those messages from that same queue. In this model, the sender knows the destination of the message and posts directly to it.
```

Reply:
Note that this text uses...:
- sender posts message
- receiver reads message
- destination
However, the following text uses...:
- consumer receives message
- sender
- consumer consumes
- message is sent
- message is processed
Can we standardize to one term for each of sender, receiver, sent, and received?